### PR TITLE
ENYO-6121: Add VGL QA sample

### PR DESCRIPTION
**Pull request by LucieRoy**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed

* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)

### CODE REVIEW

File: packages/sampler/stories/qa/VirtualGridList.js [hunk 1]
```diff
@@ -60,6 +64,102 @@ const updateDataSize = (dataSize) => {
...
+	getScrollTo = (scrollTo) => {
+		this.scrollTo = scrollTo;
+	}
+
+	render () {
+		let props = {...this.props};
+		delete props.closePopup;
+
+		return (
+			<div {...props} style={{width: '915px', height: '600px'}}>
```
Comment (webOS101) [inline packages/sampler/stories/qa/VirtualGridList.js]: Should we set this to be scaled?

File: packages/sampler/stories/qa/VirtualGridList.js [hunk 2]
```diff
@@ -60,6 +64,102 @@ const updateDataSize = (dataSize) => {
...
+
+class ButtonAndVirtualGridList extends React.Component {
+	constructor (props) {
+		super(props);
+		this.state = {
+			isPopup: false
+		};
+	}
+
+	renderPopup = ({...rest}) => {
```
Comment (webOS101) [inline packages/sampler/stories/qa/VirtualGridList.js]: can just use `(rest)` here, no need to destructure.

File: packages/sampler/stories/qa/VirtualGridList.js [hunk 3]
```diff
@@ -60,6 +64,102 @@ const updateDataSize = (dataSize) => {
...
+		this.setState({isPopup: true});
+	}
+
+	closePopup = () => {
+		this.setState({isPopup: false});
+	}
+
+	handleOpen = () => {
+		// console.log('onOpen');
+	}
```
Comment (webOS101) [inline packages/sampler/stories/qa/VirtualGridList.js]: Can delete this and the reference below.

File: packages/sampler/stories/qa/VirtualGridList.js [hunk 4]
```diff
@@ -60,6 +64,102 @@ const updateDataSize = (dataSize) => {
...
+					open={this.state.isPopup}
+					popupComponent={this.renderPopup}
+					onClick={this.openPopup}
+					direction="right"
+					showCloseButton
+					spotlightRestrict="self-only"
+					onOpen={this.handleOpen}
+					onClose={this.closePopup}
+				>
+					CAT
```
Comment (webOS101) [inline packages/sampler/stories/qa/VirtualGridList.js]: Why CAT?

Comment (LucieRoy) [reply packages/sampler/stories/qa/VirtualGridList.js]: 'Click here' is better.

File: packages/sampler/stories/qa/VirtualGridList.js (hunk 3 above)
Comment (LucieRoy) [reply packages/sampler/stories/qa/VirtualGridList.js]: Deleted

File: packages/sampler/stories/qa/VirtualGridList.js (hunk 2 above)
Comment (LucieRoy) [reply packages/sampler/stories/qa/VirtualGridList.js]: Done

File: packages/sampler/stories/qa/VirtualGridList.js (hunk 1 above)
Comment (LucieRoy) [reply packages/sampler/stories/qa/VirtualGridList.js]: Done

**Pull request merged**